migrations/sql/R__view_notice_of_work.sql

```diff
@@ -8,6 +8,7 @@ nid.now_number,
 app.lead_inspector_party_guid,
 concat_ws (' ', p.first_name, p.party_name) AS lead_inspector_name,
 COALESCE(nowt.description, sub.noticeofworktype, msub.noticeofworktype) as notice_of_work_type_description,
+atc.description as application_entity_type_description,
```

Comment:
I don't think the word "entity" is necessary since the table is called `application_type_code`. How about `application_type_description`?

Reply:
the reason for an entity because I do not want it to be confused with the different now_type_description that we already have in our system, and explicitly point out that this is a more generic type. Anyway, happy to rename if needed